src/engraving/libmscore/textbase.cpp

```diff
@@ -21,6 +21,8 @@
  */
 
 #include <cmath>
+
+#include <QGuiApplication>
```

Comment:
Perhaps it might make sense to include it only here, I assume that this was needed in the rest of the files just because of include of "textbase"?

Most of them are classes derived/related of "textbase"

Reply:
no, this is for qApp->translation
In the future should be replaced to use our `trc` function